sysdig/resource_sysdig_secure_policy.go

```diff
@@ -98,7 +98,7 @@ func resourceSysdigSecurePolicy() *schema.Resource {
 						},
 						"capture": {
 							Type:     schema.TypeList,
-							Optional: true,
+							Required: true,
```

Comment:
I believe this makes the capture block required which is not what we want. we should change the optional field on line 116 to required for just the name field